buildSrc/src/main/groovy/scripts/Utils.groovy

```diff
@@ -49,6 +49,8 @@ class Utils {
             'provider': 'gradle',
             'Implementation-Title': title,
             'Implementation-Version': "$version $revision",
+            'Bundle-Version': "$version",
```

Comment:
This is an OSGi specific attribute. We shouldn't be using it for our purposes.

Reply:
What's the alternative way to get the version from the manifest without the commit hash ? Should I create a custom entry for that ?